deployment_scripts/linode-marketplace-influxdb/influxdb-deploy.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -e
+trap "cleanup $? $LINENO" EXIT
+
+## InfluxDB Settings
+#<UDF name="soa_email_address" label="Email address (for the Let's Encrypt SSL certificate)" example="user@domain.tld">
+
+## Linode/SSH Security Settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode: *No Capital Letters or Special Characters*">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+#<UDF name="pubkey" label="The SSH Public Key that will be used to access the Linode (Recommended)" default="">
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. This is needed to create your server's DNS records" default="">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+
+# git repo
+export GIT_REPO="https://github.com/akamai-compute-marketplace/marketplace-apps.git"
+export WORK_DIR="/tmp/marketplace-apps" 
+export MARKETPLACE_APP="apps/linode-marketplace-influxdb"
+
+# enable logging
+exec > >(tee /dev/ttyS0 /var/log/stackscript.log) 2>&1
+
+function cleanup {
+  if [ -d "${WORK_DIR}" ]; then
+    rm -rf ${WORK_DIR}
+  fi
+
+}
+
+function udf {
+  local group_vars="${WORK_DIR}/${MARKETPLACE_APP}/group_vars/linode/vars"
+  local web_stack=$(echo ${WEBSERVER_STACK} | tr [:upper:] [:lower:])
+  sed 's/  //g' <<EOF > ${group_vars}
+
+  # deployment vars
+  # sudo username
+  username: ${USER_NAME}
+  webserver_stack: lemp
+EOF
+
+  if [[ -n ${SOA_EMAIL_ADDRESS} ]]; then
+    echo "soa_email_address: ${SOA_EMAIL_ADDRESS}" >> ${group_vars};
+  else echo "No email entered";
+  fi
+
+  if [[ -n ${PUBKEY} ]]; then
+    echo "pubkey: ${PUBKEY}" >> ${group_vars};
+  else echo "No pubkey entered";
+  fi
```

Comment:
remove pubkey var

Reply:
removed in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)